src/components/subscriptions/tests/licenses/LicenseAllocationDetails.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import {
+  beforeEach,
+  describe,
+  jest,
+  test,
+} from '@jest/globals';
```

Comment:
Wondering about why `@jest/globals` is needed since they should already be globals?